codegenerator/cli/npm/envio/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "homepage": "https://envio.dev",
   "dependencies": {
-    "rescript": "11.1.3"
+    "rescript": "11.1.3",
+    "rescript-schema": "8.2.0"
```

Comment:
We should keep the `package.json.tmpl` dependencies in sync. Need to add `rescript-schema` there as well

Reply:
Ah good spot 👌🏼